### DISCUSSION TITLE
Mirror from one discord channel and send it to multiple servers channels

### DISCUSSION THREAD

**Discussion by cs-devansh-mahajan** [Rapptz/discord.py#9029] [Q&A] [answered]
Hey so I wanted to create a bot which would mirror my message from one channel to different discord channels using embeds is there a way to do this and send embeds with javascript in python code?

**Comment by DenverCoder1** [comment, marked as answer]
Your best option would be to create a webhook to each of the channels and send the embed to each webhook.

You can create the embeds with [`discord.Embed`](https://discordpy.readthedocs.io/en/stable/api.html#discord.Embed) and send it using the `embed` or `embeds` kwarg.

You propbably also want to use an `on_message` event override or listener to detect incoming messages and perform the webhook sending there.

The quickstart and docs might help if you are new to Discord.py - https://discordpy.readthedocs.io/en/stable/quickstart.html

You can also ask on the Discord.py Discord server or search for things that may have been asked before.
